bin/ci

```diff
@@ -84,7 +84,9 @@ prepare_system() {
 
 build() {
   with_build_env 'make std_spec clean'
-  with_build_env 'make crystal spec doc'
+  with_build_env 'make crystal doc'
+  on_linux with_build_env 'make spec coverage_output=.build/coverage'
+  on_osx with_build_env 'make spec'
```

Comment:
Installing kcov on osx is pretty hard and would require compiling kcov every build. Is it worth it when the coverage on linux is likely to be the same? I think so as kcov is actually a really useful test of the debug information's accuracy.